libs/blocks/global-navigation/base.css

```diff
@@ -115,14 +115,51 @@
     padding: 0 12px;
   }
 
+  .feds-navItem--active > .feds-navLink {
+    position: relative;
+    font-weight: 700;
```

Comment:
In the AEM implementation we're highlighting the active element with **bold**, but I see this does not get applied in Milo.

AEM:
<img width="416" alt="Screenshot 2023-12-06 at 14 10 32" src="https://github.com/adobecom/milo/assets/408175/e88f6555-4144-4711-b727-390b58b8cd20">

Milo:
<img width="419" alt="Screenshot 2023-12-06 at 14 11 38" src="https://github.com/adobecom/milo/assets/408175/dfabe2c8-aa64-4c45-961c-95ed808d6bde">

Reply:
I'm fine with keeping things as is as long as everything has been discussed. Maybe we should add a note in the JIRA ticket for future reference.